config/default.config

```diff
@@ -5,6 +5,8 @@ torture.generator.amo       true
 torture.generator.mul       true
 torture.generator.divider   true
 torture.generator.run_twice true
```

Comment:
Isn't this run_twice config obsolete now (as I recall it is basically a hardcoded loop_size=2)